docs/source/reference/respawn/hud_element_notation.rst

```diff
@@ -3,6 +3,8 @@
 Element Notation
 ====
 
+Elements are notated as `VDF<https://developer.valvesoftware.com/wiki/KeyValues>`_
```

Comment:
Link got scuffed in render ^^

![image](https://user-images.githubusercontent.com/40122905/202559412-6a3ddf0d-5c24-4f00-a77f-af7bb73edac8.png)

Reply:
Confirmed fixed in 8094bbd0ee7fbdf4ec2a5999314ba8173f113575